common/common.go

```diff
@@ -87,6 +87,21 @@ func HexDecodeInto(dst []byte, h []byte) error {
 	return nil
 }
 
+// Uint8 returns a byte array from a uint8
+func Uint16ToBytes(u uint16) []byte {
+	buff := new(bytes.Buffer)
```

Comment:
because with [4]byte will give bad result :yum: . Now serious, because was how Uint32ToBytes was implemented, but no problem, I've changed now to the `binary.LittleEndian.PutUint16` version